app/conversations/__init__.py

```diff
@@ -32,3 +32,10 @@ def post(self, user=None):
         
         result = SocketService.create_conversation(conversation_id, user['email'], participants)
         return utils.make_response(data = { 'conversationId': conversation_id, 'participants': result })
+
+    def remove(self, user=None):
```

Comment:
The name of methods in a `Resource` must all be HTTP verbs. I think an appropriate verb for this one could be `put`.